exotic/api/output_aavso.py

```diff
@@ -112,7 +112,6 @@ def aavso(self, airmasses, ld0, ld1, ld2, ld3, tmidstr):
                     f"#PRIORS-XC={dumps(priors_dict)}\n"  # code yields
                     f"#RESULTS=Tc={round_to_2(self.fit.parameters['tmid'], self.fit.errors['tmid'])} +/- {round_to_2(self.fit.errors['tmid'])}"
                     f",Rp/R*={round_to_2(self.fit.parameters['rprs'], self.fit.errors['rprs'])} +/- {round_to_2(self.fit.errors['rprs'])}"
-                    f",a/R*={round_to_2(self.fit.parameters['ars'], self.fit.errors['ars'])} +/- {round_to_2(self.fit.errors['ars'])}"
```

Comment:
why did you remove a/Rs here? because we don't fit for it explicitly?

Reply:
Yes, a/Rs is not always a free parameter and this line of code will error out. I've switched to fitting for inclination in order to estimate the transit duration

Reply:
this line was causing that try/except statement above to trigger the except due to my comment above. It's not necessary to include it since we already list the value in the priors.